TribalWars/src/WebApi/Controllers/TribeController.cs

```diff
@@ -86,5 +86,14 @@ public async Task<ActionResult> TribeMembers([FromHeader] int tribeId)
                 ? Ok(result)
                 : BadRequest(result);
         }
+
+        [HttpDelete("/disband-tribe")]
+        public async Task<ActionResult> DisbandTheTribe([FromBody] DisbandTribeDto dto)
```

Comment:
DisbandTheTribe or DisbandTribe? Be consistent :)

Reply:
Ok, I changed to DisbandTribe